.kokoro/presubmit/cloudsql.sh

```diff
@@ -39,7 +39,7 @@ export NVM_DIR="$HOME/.nvm"
 [ -s "$NVM_DIR/nvm.sh" ] && \. "$NVM_DIR/nvm.sh"  # This loads nvm
 
 # Use Node 8
```

Comment:
Would you mind updating these comments as well?

Reply:
My bad; missed that part.